backend/spec/graphql/types/query_type_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'journals' do
+    let(:query) do
+      <<~QUERY
+        query {
```

Comment:
こちらに機能解説できなかったことを調べてみました！
https://dmmwebcamp-talk.slack.com/archives/C04MD1KN797/p1690149375944399

ここに従うと、以下の方が良さそうかなと思いました！
```suggestion
        query GetAllJournals{
```

Reply:
LGTM